zcash/zcash
# Initial packaging for Debian

This adds some source files, a script and skeleton for building a Debian package that installs the zcash daemon and CLI tool.

To test it:

Check out my branch.
Build Zcash binaries: `./zcutil/build.sh`
Build the package: `./zcutil/build-debian-package.sh`
You now have a .deb in the source root folder, so install it: `sudo dpkg -i zcash-1.0.0-rc1-amd64.deb`
Now you can run `zcashd`, `zcash-cli`, and `zcash-fetch-params` from `$PATH`.

This pull request will need further refinement and review, so let's leave it open for a while.
